Add a note for light-dark to "Styling console output" section (#34505)
mdn/content: files/en-us/web/api/console/index.md

- {{cssxref("word-spacing")}} and {{cssxref("word-break")}}
- {{cssxref("writing-mode")}}

> > **Note:** Each console message behaves like an inline element by default. If you want to properties such as `padding`, `margin`, and so to have any effect, you can set the `display` property to `display: inline-block`.

Reviewer: ```suggestion
> **Note:** Each console message behaves like an inline element by default. If you want properties such as `padding`, `margin`, and so on to have any effect, you can set the `display` property to `display: inline-block`.
```

> > **Note:** The console message behaves like an inline element by default. To see the effects of `padding`, `margin`, etc. you should set it to for example `display: inline-block`.

Reviewer: ```suggestion
> **Note:** Each console message behaves like an inline element by default. If you want to properties such as `padding`, `margin`, and so to have any effect, you can set the `display` property to `display: inline-block`.
```

> > **Note:** In order to support both light and dark color schemes, {{cssxref("color_value/light-dark")}} can be used when specifying color. e.g. `color: light-dark(#D00000, #FF4040);`

Reviewer: ```suggestion
> **Note:** In order to support both light and dark color schemes, {{cssxref("color_value/light-dark")}} can be used when specifying colors; for example: `color: light-dark(#D00000, #FF4040);`
```
